loadtest/control/gencontroller/controller.go

```diff
@@ -104,7 +105,7 @@ func (c *GenController) Run() {
 
 	for i := 0; i < len(initActions); i++ {
```

Comment:
This doesn't retry the action right? We probably need to break this out of the loop and call this via `c.runActions`.

Reply:
Init actions are executed again if they error as the loop counter is decremented so it should be okay I am thinking.

Reply:
Ah yes! Missed the `i--` :)